### PR TITLE
[RFR] Full sample config.

### PR DESCRIPTION
**Pull request by manasdk**
Tool to generate complete config and generate config file.

### CODE REVIEW

File: conf/st2.full.conf [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+serve_webui_files = True
+# StackStorm API server host
+host = 0.0.0.0
+# Send empty message every N seconds to keep connection open
+heartbeat = 25
+# StackStorm API server port
+port = 9101
+
+[api_pecan]
+# None
```
Comment (Kami) [inline conf/st2.full.conf]: Hm, seems about to add description for all the arguments which are missing it :)

**Comment by dzimine** [comment]
Good. suggest we call it st2.conf.sample

File: conf/st2.full.conf (hunk 1 above)
Comment (manasdk) [inline conf/st2.full.conf]: I am going to be lazy and skip the entire api_pecan section. Not sure if there is much value in advertising that group.

f78b3836ab7e001618f01d0fae48f8236ca8fd8e

**Comment by manasdk** [comment]
> suggest we call it st2.conf.sample

Fixed in 09b867bae0a5f7e6f1418fa8e7b6690b1c89b914

File: conf/st2.full.conf (hunk 1 above)
Comment (Kami) [inline conf/st2.full.conf]: I'm fine with that.

**Comment by Kami** [comment]
LGTM, +1

**Comment by Kami** [comment]
Perhaps also reference this sample config with all the options somewhere in the docs?

**Comment by manasdk** [comment]
docs updated : 25d736560fa013707e7f8744490af22d50d2882f

**Comment by Kami** [comment]
:+1:

**Comment by jfryman** [comment]
👍

**Pull request merged**